### PR TITLE
Fix some Bitcoin mentions in Qt

### PR DESCRIPTION
**Pull request by stevenroose**
Solves https://github.com/ElementsProject/elements/issues/667.

### CODE REVIEW

**Comment by greenaddress** [comment]
utack 8e02d7ee6953d134af80b1f6ce25d340c61533eb

**Comment by wintercooled** [comment]
Looks good to me

File: src/qt/bitcoin.cpp [hunk 1]
```diff
@@ -390,7 +390,7 @@ void BitcoinApplication::shutdownResult()
 
 void BitcoinApplication::handleRunawayException(const QString &message)
 {
-    QMessageBox::critical(nullptr, "Runaway exception", BitcoinGUI::tr("A fatal error occurred. Bitcoin can no longer continue safely and will quit.") + QString("\n\n") + message);
+    QMessageBox::critical(nullptr, "Runaway exception", BitcoinGUI::tr("A fatal error occurred. Elements can no longer continue safely and will quit.") + QString("\n\n") + message);
```
Comment (philippem) [inline src/qt/bitcoin.cpp]: This is a key that is used to index into the translation. If we change the string here without changing the <source> elements in the locale specific translations, the localized builds may fail to display a string.
For example:

https://github.com/ElementsProject/elements/blob/master/src/qt/locale/bitcoin_es.ts#L576

**Comment by philippem** [comment]
utNACK
If we change a string key we need to at least change the same key in the locale files

File: src/qt/bitcoin.cpp (hunk 1 above)
Comment (stevenroose) [reply src/qt/bitcoin.cpp]: Yeah it seemed that that's what we have been doing in the past.

**Comment by stevenroose** [comment]
@philippem  Hmm, I think we discussed this in a private convo, can you ACK that you agree? Since we never updated any of the translation files in the past and just left them for what they are, this PR is not the place to do that. If we want to support localization, we should have a more wide effort for porting all the missing strings.

Fyi, `rg -i elements src/qt/locale` shows nothing (except for one phrase that just has the common word elements in it in a single language).

**Comment by stevenroose** [comment]
@philippem If you ACK this one, can you also ACK the backport: https://github.com/ElementsProject/elements/pull/848.

**Comment by achow101** [comment]
The second commit is unnecessary. Qt will fall back to the text in the source code if it is unable to find the string in the translation files. Old strings in the translation files will be removed during a translation file update.

File: src/qt/intro.cpp [hunk 2]
```diff
@@ -130,7 +130,7 @@ Intro::Intro(QWidget *parent, uint64_t blockchain_size, uint64_t chain_state_siz
         .arg(tr(PACKAGE_NAME))
         .arg(m_blockchain_size)
         .arg(2009)
```
Comment (philippem) [inline src/qt/intro.cpp]: this should be 2018 which is when the liquidv1 chain started

**Comment by stevenroose** [comment]
> The second commit is unnecessary. Qt will fall back to the text in the source code if it is unable to find the string in the translation files. Old strings in the translation files will be removed during a translation file update.

@achow101 Well, I do a bunch of actual translation updates. (Where it's just changing the Bitcoin->Elements name.) So in those cases, it should be slightly better, right?

File: src/qt/intro.cpp (hunk 2 above)
Comment (stevenroose) [reply src/qt/intro.cpp]: I updated it to 2016 (the start of Elements Alpha) in an extra commit.

**Comment by achow101** [comment]
> @achow101 Well, I do a bunch of actual translation updates. (Where it's just changing the Bitcoin->Elements name.) So in those cases, it should be slightly better, right?

It won't affect anything and potentially makes the automated stuff break. I've found that manually tinkering with translation files is a good way to have a bad day.

**Comment by stevenroose** [comment]
> It won't affect anything and potentially makes the automated stuff break. I've found that manually tinkering with translation files is a good way to have a bad day.

I don't understand what you mean here. You're saying that I should just not touch the translation files at all, even if it's trivial to change (both the `<source>` and `<translation>` parts)?

@philippem so I just remove that commit _again_?

**Comment by achow101** [comment]
> I don't understand what you mean here. You're saying that I should just not touch the translation files at all, even if it's trivial to change (both the `<source>` and `<translation>` parts)?

Yes. If there is no plan to support translations, then there's no point in touching the files. If there is a plan to support translations, then automated scripts will update and clean up everything for you. There's no reason to change the translation files as any manual changes will get cleaned up by other processes. It just adds noise to the diff.

**Pull request merged**